master/docs/manual/cfg-buildsteps.rst

```diff
@@ -2443,6 +2443,14 @@ Hyperlinks are added to the build detail web pages for each triggered build.
 
         It is possible, but not advisable, to create a cycle where a build continually triggers itself, because the schedulers are specified by name.
 
+``unimportantSchedulerNames``
+    When ``waitForFinish`` is ``True``, all schedulers in this list will not cause the trigger step to fail. unimportantSchedulerNames must be a subset of schedulerNames
+    If ``waitForFinish`` is ``False``, unimportantSchedulerNames will simply be ignored.
+    
+.. note::
+
+        ``schedulerNames`` must have at least one scheduler not present in ``unimportantSchedulerNames``
```

Comment:
you're right. After switching to super set SchedulerNames and unimportantSchedulerNames can actually match.